script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -126,97 +144,112 @@ I'm ashamed of how forgetful I am.
 [sha:f129ec04d24ffa9c51ec2338cb045b350a7994c9]{
 -- Page 595, Offset 28738.
 -- 「それは仕方のない事です。昔の遠野邸はご来賓の方も多かったですし、なにより子供の頃の話ですから。
-"You're not at fault. There were many guests visiting the Tohno mansion back then, and above all, it was a long time ago when we were children.
+"It's okay, I can't blame you.
+ There were a lot of people coming and going at the Tohno mansion back then,
+ and besides, it was back when we were children.
 }
 [sha:65e726a2d299d428902e73528c92766777b4eff3]{
 -- Page 595, Offset 28739.
 -- 今ですらこんな大きなお屋敷なんですよ？　子供の視点では果てのない迷路のように思えたものです。この本館だけでも覚えきれなかったのに、今はもう取り壊されたテニスコート横の迎賓館や植物園、離れのお屋敷もありました。
-Even now, you still see it as a big mansion, don't you? To a child, it must have looked like an endless maze. You never completely familiarized yourself with the main mansion building, let alone the guest house and botanical garden near what was the now demolished tennis. There was also the detached mansion building.
+It still feels like such a big mansion even now, doesn't it?
+ To a child, it seemed like an endless maze.
+ I could barely remember my way around the main building,
+ let alone the guest house and botanical garden next to where the tennis courts used to be,
+ or the detached mansion building.
 }
 [sha:eb056550467c7d1570804c0fc23f2289bae6021d]{
 -- Page 595, Offset 28740.
 -- ご親戚の方々も滞在していましたし、あの頃、誰がどこにいたのかを把握していたのは、それこそご当主であらせられた槙久様だけかと存じます」
-Many relatives stayed here as well. I imagine that only Makihisa-sama as the master of the family knew exactly the ins and outs of those arrangements."
+There were a lot of relatives staying here as well.
+ I imagine only Makihisa-sama ever knew who and where everyone was at any given point in time."
 }
 [sha:2d79e91b73430fdc93e063781cd7ea629cfee6b3]{
 -- Page 596, Offset 28741.
 -- 「……そうですね。言われてみれば、確かに……」
-"Right... Now that you say it, it does ring a bell..."
+"Oh yeah...
+ You've jogged my memory.
+ In fact, if I think about it..."
 }
 [sha:5ddc808497198e766b29dc97ec80b7794a456156]{
 -- Page 597, Offset 28742.
 -- 子供の時分は危ないからひとりで二階には行かないようにとか、迎賓館には近づくなとか叱られた覚えがある。
-I remember as a child being told not to go to second floor because it was dangerous, or being scolded because I got too close to the guest house.
+I remember being warned never to go to the second floor alone as a child because it was dangerous,
+ and getting scolded for getting too close to the guest house.
 }
 [sha:0fb715000b2ecd5a8dec1237b457b1b8e4090db9]{
 -- Page 597, Offset 28743.
 -- これだけ広い屋敷なんだから、顔を合わせる機会がないまま生き別れた誰かがいたのかもしれない。
-Since the mansion is so vast, there could have been a relative who lived apart and whom I've never had the opportunity to meet.
+In a mansion this large, it's pretty plausible that I could have gone the whole time living here without running into someone.
```

Comment:
the way this sentence is phrased makes it sound like he'd never meet anyone period.

>...it's pretty plausible that there were people I didn't run into, despite having lived here my whole life.